src/Microsoft.VisualStudio.Web.CodeGeneration.Tools/project.json

```diff
@@ -26,37 +28,33 @@
     }
   },
   "dependencies": {
-    "Microsoft.DotNet.Cli.Utils": "1.0.0-preview3-*",
-    "Microsoft.Extensions.CommandLineUtils": "1.1.0-*",
-    "Microsoft.Extensions.DotnetToolDispatcher.Sources": {
-      "version": "1.1.0-*",
-      "type": "build"
-    },
-    "Microsoft.Extensions.PlatformAbstractions": "1.1.0-*",
-    "Microsoft.Extensions.FileProviders.Physical": "1.1.0-*",
-    "Microsoft.DotNet.InternalAbstractions": "1.0.0-*",
+    "Microsoft.DotNet.Cli.Utils": "1.0.0-preview2-003121",
```

Comment:
:watch: Depending on the response we get on the internal email thread, this may be a behavior we have to build ourselves if we are not allowed to use the preview3 version of the API.